superset-frontend/src/dashboard/components/nativeFilters/utils.test.ts

```diff
@@ -45,6 +45,12 @@ describe('nativeFilterGate', () => {
     });
 
     it('should return false for native filter chart with cross filter support', () => {
+      isFeatureEnabledMock.mockImplementation(feature => {
+        if (feature === FeatureFlag.DASHBOARD_CROSS_FILTERS) return false;
```

Comment:
I believe `FeatureFlag.DASHBOARD_CROSS_FILTERS` will not exist anymore given that we are removing the feature flag. We need to remove all references of it.